Pinta.Core/Managers/PaletteManager.cs

```diff
@@ -132,6 +135,14 @@ private void AddRecentlyUsedColor (Color color)
 			OnRecentColorsChanged ();
 		}
 
+		private void PopulateSavedPalette ()
+		{
+			var palette_file = System.IO.Path.Combine (PintaCore.Settings.GetUserSettingsDirectory (), "palette.txt");
```

Comment:
Could you add a constant for `palette.txt`, similar to `SETTINGS_FILE` in the other file?